lib/tests/test.js

```diff
@@ -117,16 +117,23 @@ class Test {
     this.events.once('ready', () => {
       callback();
     });
+    this.events.once('deployError', (err) => {
+      callback(err);
```

Comment:
Wouldn't that cause callback already called at some point? Like one test passes, then the second fails, it triggers that event and the previous callback gets called again? Maybe there is a way to remove the listener on the other when one of the events is met?